war/src/main/less/modules/icons.less

```diff
@@ -0,0 +1,31 @@
+@import '../abstracts/colors.less';
```

Comment:
I am surprised that there is not class for that already. Maybe it is a part of icon-shim?

Reply:
class for importing our colours?

Reply:
Ideally we would have classes like bootstrap `.text-primary`, `.text-secondary`, `.text-bold` etc. The reason I haven't tried to implement them yet is because I think our colour palette use is not mature enough, and probably won't be for a while, so IMO we lack some information to do it properly.